hackathon_site/dashboard/frontend/src/slices/hardware/cartSlice.ts

```diff
@@ -49,6 +94,20 @@ const cartSlice = createSlice({
             cartAdapter.updateOne(state, payload);
         },
     },
+    extraReducers: (builder) => {
+        builder.addCase(submitOrder.pending, (state) => {
+            state.isLoading = true;
+            state.error = null;
+        });
+        builder.addCase(submitOrder.fulfilled, (state) => {
+            state.isLoading = false;
+            state.error = "Submitted";
```

Comment:
Should set this to null. Eventually for handling errors on the frontend, we'll look to see if it's null or not.